src/pks/mpc/mpc_delegate_water.cc

```diff
@@ -436,11 +436,11 @@ MPCDelegateWater::ModifyPredictor_TempFromSource(double h, const Teuchos::RCP<Tr
 
     const Epetra_MultiVector& Told = *S_inter_->GetFieldData("surface_temperature")
         ->ViewComponent("cell",false);
-    const Epetra_MultiVector& Tsource = *S_next_->GetFieldData("surface_mass_source_temperature")
+    const Epetra_MultiVector& Tsource = *S_next_->GetFieldData("surface_water_source_temperature")
```

Comment:
Haha, this has clearly not been used in a very long time.  It should probably get deleted.  All of these hard-coded names have changed, e.g. "suface-temperature", "surface-water_source_temperature", etc.

Reply:
I'll deal with this, don't worry about this comment for your pull request.

Reply:
#89